From: Adrien Plazas <kekun.plazas@laposte.net>
Date: Sat, 13 Jul 2019 10:39:21 +0200
Subject: infobar: Move the action area below

This will help the infobars fit on phone screens.

Because of how GtkInfoBar API works, this cannot really be made adaptive
without breaking it in the process. So, desktop will have to live with
funny infobars for now.
---
 gtk/ui/gtkinfobar.ui | 28 ++++++++++++++--------------
 1 file changed, 14 insertions(+), 14 deletions(-)

diff --git a/gtk/ui/gtkinfobar.ui b/gtk/ui/gtkinfobar.ui
index 0b0eaf1..9463c64 100644
--- a/gtk/ui/gtkinfobar.ui
+++ b/gtk/ui/gtkinfobar.ui
@@ -6,6 +6,10 @@
       <object class="GtkRevealer" id="revealer">
         <property name="visible">1</property>
         <property name="reveal-child">1</property>
+        <child>
+          <object class="GtkBox">
+            <property name="visible">1</property>
+            <property name="orientation">vertical</property>
         <child>
           <object class="GtkBox" id="content">
             <property name="visible">1</property>
@@ -20,17 +24,6 @@
                 <property name="expand">1</property>
               </packing>
             </child>
-            <child>
-              <object class="GtkButtonBox" id="action_area">
-                <property name="visible">1</property>
-                <property name="border-width">6</property>
-                <property name="spacing">6</property>
-                <property name="layout-style">end</property>
-              </object>
-              <packing>
-                <property name="position">1</property>
-              </packing>
-            </child>
             <child>
               <object class="GtkButton" id="close_button">
                 <property name="can-focus">1</property>
@@ -47,9 +40,16 @@
                   </object>
                 </child>
               </object>
-              <packing>
-                <property name="position">2</property>
-              </packing>
+            </child>
+          </object>
+        </child>
+            <child>
+              <object class="GtkButtonBox" id="action_area">
+                <property name="visible">1</property>
+                <property name="border-width">6</property>
+                <property name="layout-style">expand</property>
+                <property name="orientation">vertical</property>
+              </object>
             </child>
           </object>
         </child>
